core/types/transaction_signing.go

```diff
@@ -27,7 +27,12 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
-var ErrInvalidChainId = errors.New("invalid chaid id for signer")
+var (
+	ErrInvalidChainId = errors.New("invalid chaid id for signer")
+
+	errAbstractSigner     = errors.New("abstract signer")
+	abstractSignerAddress = common.HexToAddress("ffffffffffffffffffffffffffffffffffffff")
```

Comment:
You have `ffffffffffffffffffffffffffffffffffffff`, that's only 38 characters. Should be 40, right ? 
https://github.com/ethereum/EIPs/issues/86